test/unit/models_test.rb

```diff
@@ -21,7 +21,7 @@ class ModelsTest < ActiveSupport::TestCase
       'LatestForumPostsPortlet' => %w[content_type], 'TableOfContentsPortlet' => %w[content_type], 'AuthenticationProvider::GitHub' => %w[branding_state account_type],
       'AuthenticationProvider::Keycloak' => %w[account_type], 'AuthenticationProvider::Auth0' => %w[account_type], 'AuthenticationProvider::Custom' => %w[account_type],
       'AuthenticationProvider::ServiceDiscoveryProvider' => %w[account_type], 'AuthenticationProvider::RedhatCustomerPortal' => %w[account_type],
-      'Account' => %w[credit_card_auth_code credit_card_authorize_net_payment_profile_token credit_card_partial_number]
+      'Account' => %w[credit_card_auth_code credit_card_partial_number credit_card_authorize_net_payment_profile_token]
```

Comment:
Shouldn't we remove this reference instead of reordering it?

Reply:
maybe part of another PR where migrations are done?

Reply:
The reorder is wrong, it should be unchanged. As @akostadinov said, it has to be removed after the migration, otherwise the model validation test fails: https://app.circleci.com/pipelines/github/3scale/porta/27457/workflows/c8b186c0-5b02-4afc-9302-d29698a54e3e/jobs/307709/tests